source/common/runtime/runtime_impl.h

```diff
@@ -190,6 +185,11 @@ class NullLoaderImpl : public Loader {
       return default_value;
     }
 
+    std::unordered_map<std::string, const Snapshot::Entry*> getAll() const override {
+      std::unordered_map<std::string, const Snapshot::Entry*> entries;
```

Comment:
Could do just `return std::unordered_map<std::string, const Snapshot::Entry*>();` - your call

Reply:
I'll go with that. Much cleaner :)